File: JAVA/Daily Flash Codes/Week2/Day4_Jan23/code/prog2.java

import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
 		System.out.print("Enter row num: ");
		int row = Integer.parseInt(in.readLine());
		int var =1;

		for(int olc = 0; olc<row; olc++) {
			for(int ilc = 0; ilc<= olc; ilc++) 
				System.out.print(var++ + " ");		

			System.out.println();
		}
	}
}

File: C/Daily Flash/Week2/Day4_Jan23/code/prog2.c
/*
 rogram 2: Write a Program to Print following Pattern
1
2 3 
4 5 6 
7 8 9 10
*/

#include <stdio.h>

void main () {
	int row, var=1;
	printf("Enter row: ");
	scanf("%d",&row);
	
	for(int olc = 0; olc<row; olc++) {
		
		for(int ilc = 0; ilc<=olc; ilc++) {
			printf(" %d ",var++);
		}
		printf("\n");

	}
}
